setup.cfg

```diff
@@ -43,6 +43,7 @@ install_requires =
 auth =
     baseapp-django-trench >= 0.4.4
     djangorestframework-simplejwt[crypto] >= 5.2, < 5.6
+    django-allauth == 65.13.0
```

Comment:
Shouldn't we also add this new dependency to the testproject/requirements.txt?

Reply:
I noticed we are also missing other updates in the testproject, should be similar to what you did here: https://bitbucket.org/silverlogic/baseapp-backend-template/pull-requests/224/diff